src/lib/support/tests/TestTestPersistentStorageDelegate.cpp

```diff
@@ -38,62 +38,67 @@ void TestBasicApi(nlTestSuite * inSuite, void * inContext)
     CHIP_ERROR err;
     memset(&buf[0], 0, sizeof(buf));
     size = sizeof(buf);
-    err  = storage.SyncGetKeyValue("roboto", &buf[0], size);
+    DefaultStorageKeyAllocator roboto;
+    roboto.GroupDataCounter();
+    err = storage.SyncGetKeyValue(roboto, &buf[0], size);
     NL_TEST_ASSERT(inSuite, err == CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND);
     NL_TEST_ASSERT(inSuite, size == sizeof(buf));
 
-    err = storage.SyncDeleteKeyValue("roboto");
+    err = storage.SyncDeleteKeyValue(roboto);
```

Comment:
This changes the semantics of the test that is meant to allow arbitrary key storage to be tested. This cannot remain named "roboto"